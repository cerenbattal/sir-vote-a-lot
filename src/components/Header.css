.header {
    font-size: 48px;
    color: #FFFFFF;
    padding: 0.5em 0.5em 0 0.5em;
    text-align: center;
}

h1 {
    display: inline;
    font-family: 'Pattaya', sans-serif;
    font-weight: 100;
}
