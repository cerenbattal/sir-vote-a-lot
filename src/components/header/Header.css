.header {
    font-size: 48px;
    color: #FFFFFF;
    padding: 0.5em 0.5em 0 0.5em;
    text-align: center;
    text-shadow: 0 0 3px rgba(0,0,0,0.2);
}

h1 {
    display: inline;
    font-family: 'Pattaya', sans-serif;
    font-weight: 100;
}
