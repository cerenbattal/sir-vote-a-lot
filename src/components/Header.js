import React from 'react'
import './Header.css'

export default function Header() {
    return (
        <div className="header">
            <i className="edit outline icon"></i><h1>Sir vote-a-lot</h1>
        </div>
    )
}
