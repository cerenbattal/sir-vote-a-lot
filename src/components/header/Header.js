import React from 'react'
import './Header.css'

export default function Header() {
    return (
        <div data-testid="header-test" className="header">
            <i className="edit outline icon"></i><h1>Sir vote-a-lot</h1>
        </div>
    )
}
