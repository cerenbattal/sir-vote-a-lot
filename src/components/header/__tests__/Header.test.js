import { render, screen, cleanup } from '@testing-library/react'
import renderer from 'react-test-renderer'
import '@testing-library/jest-dom/extend-expect'
import Header from '../Header'

afterEach(() => {
    cleanup();
})

test('should render header component', () => {
    render(<Header />);
    const headerElement = screen.getByTestId('header-test');
    expect(headerElement).toBeInTheDocument();
})

test('matches snapshot', () => {
    const tree = renderer.create(<Header />);
    expect(tree).toMatchSnapshot();
})