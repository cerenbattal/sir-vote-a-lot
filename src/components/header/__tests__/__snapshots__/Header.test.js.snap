// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="header"
  data-testid="header-test"
>
  <i
    className="edit outline icon"
  />
  <h1>
    Sir vote-a-lot
  </h1>
</div>
`;
